lib/csv_to_popolo.rb

```diff
@@ -339,30 +353,72 @@ def given?(key)
       @r.key?(key) && !@r[key].nil? && !@r[key].empty?
     end
 
+    def split_cell(key)
+      separator = MODEL[key][:multivalue_separator]
+      if separator
+        @r[key].split(separator)
```

Comment:
It might be worth stripping the whitespace at this point as well. I can imagine people putting whitespace before, after or even around the separator in CSV fields.